tools/scalar-schema/README.md

```diff
@@ -22,7 +22,7 @@ $ java -jar target/scalar-schema.jar --cosmos -h <YOUR_ACCOUNT_URI> -p <YOUR_ACC
 
 ```console
 # For Cassandra
-$ java -jar target/scalar-schema.jar --cassandra -h <CASSANDRA_IP> -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f schema.json [-n <NETWORK_STRATEGY> -R <REPLICATION_FACTOR>]
+$ java -jar target/scalar-schema.jar --cassandra -h <CASSANDRA_IP> -P <CASSANDRA_PORT> -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f schema.json [-n <NETWORK_STRATEGY> -R <REPLICATION_FACTOR>]
```

Comment:
Based on the suggestions from Yuji, `-P` should be optional?

Reply:
Yes, and probably `-u` and `-p` could also be optional since they have default values (`cassandra`/`cassandra`)? What do you think? @yito88

Reply:
Yes. These default values are set in `cassandra_schema.clj` when we don't specify them by the options.

Reply:
Thanks! Updated in 6d24892. (Brackets are changed to surround each optional param.)